pkg/cmd/oauth-apiserver/cmd.go

```diff
@@ -49,7 +49,6 @@ func NewOAuthAPIServerOptions(out io.Writer) *OAuthAPIServerOptions {
 		TokenValidationOptions: tokenvalidationoptions.NewTokenValidationOptions(),
 		Output:                 out,
 	}
-	o.RecommendedOptions.Etcd.StorageConfig.Paging = true
```

Comment:
why the change ?

Reply:
ok, the param was removed in https://github.com/kubernetes/kubernetes/pull/121390